test/built-ins/Function/prototype/arguments/prop-desc.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2024 Justin Dorfman. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-function.prototype.arguments
+description: >
+  Function.prototype.arguments property descriptor
+info: |
+  Function.prototype.arguments is an accessor property whose set and get
+  accessor functions are both %ThrowTypeError%.
```

Comment:
`sec-function.prototype.arguments` does not appear to be a valid anchor: https://tc39.es/ecma262/#sec-function.prototype.arguments
```suggestion
esid: sec-addrestrictedfunctionproperties
description: >
  Function.prototype.arguments is an accessor property whose set and get
  functions are both %ThrowTypeError%.
info: |
  2. Let _thrower_ be _realm_.[[Intrinsics]].[[%ThrowTypeError%]].
  3. Perform ! DefinePropertyOrThrow(_F_, *"caller"*, PropertyDescriptor { [[Get]]: _thrower_, [[Set]]: _thrower_, [[Enumerable]]: *false*, [[Configurable]]: *true* }).
  4. Perform ! DefinePropertyOrThrow(_F_, *"arguments"*, PropertyDescriptor { [[Get]]: _thrower_, [[Set]]: _thrower_, [[Enumerable]]: *false*, [[Configurable]]: *true* }).
```